### PR TITLE
Updated read_hifld_data, fixed package imports

### PR DESCRIPTION
**Pull request by erika-tyagi**
* Updated `read_hifld_data` to output lat / lon coordinates (I changed my mind and ended up using the centroid because it was pretty fast and seemed more robust and deterministic) - I think we can also eventually host this on our server instead of reading from the HIFLD website every time? 
* Moved the `readr::cols()` import to `clean_facility_name` since `read_hifld_data` no longer used it 
* Added some more apt addons so the CI wouldn't yell at me

### CODE REVIEW

File: R/read_hifld_data.R [hunk 1]
```diff
@@ -1,25 +1,44 @@
...
+#' coordinates (latitude and longitude) for the centroid of each facility.
 #'
-#' @return data frame with facility info
+#' @return data frame with facility info and coordinates 
 #'
-#' @importFrom readr read_csv
-#' @importFrom readr cols
-#' @importFrom utils download.file
-#' @importFrom dplyr rename
+#' @import dplyr
```
Comment (nmmarquez) [inline R/read_hifld_data.R:8]: Can we add an example usage here?

**Review by hjohns12 — APPROVED** [review]
Looks good to me!

File: R/read_hifld_data.R (hunk 1 above)
Comment (erika-tyagi) [reply R/read_hifld_data.R:8]: Good call, done!

**Comment by nmmarquez** [comment]
This just might be a personal set up issue but I get errors and the test build listing the following

```
 Error: Problem with `mutate()` input `coords`.
x OGRCreateCoordinateTransformation() returned NULL: PROJ available?
ℹ Input `coords` is `sf::st_centroid(sf::st_transform(geometry, 2901))`
```

Can you link what version of PROJ your system is using? Im on 6.3.2

**Comment by erika-tyagi** [comment]
Yay for teamwork on this one! I think it's ready to go, but one last confirmation on @nmmarquez's machine would make me feel better!

**Pull request merged**